apps/content-script/src/components/ShowWordCardWrapper.svelte

```diff
@@ -92,7 +92,7 @@
         on:rightAnswer={() => onAnswer(true)}
         on:wrongAnswer={() => onAnswer(false)}
         options={_options}
-        showWordDurationSeconds={_settings?.showCardDurationSeconds ?? 500}
+        showWordDurationSeconds={_settings?.showMCQDurationSeconds ?? 500}
```

Comment:
500 as a default value seems very high. Preferably use 16 as in the default user settings in 
`apps/backend-functions/src/app/user-management.ts` => line 36